src/App.js

```diff
@@ -1,6 +1,8 @@
 import React, { useState } from 'react';
 import './App.scss';
 import data from './data/dummy.json';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
The bootstrap CSS was already imported in `index.js` so we can remove this here

Reply:
Yes, done.